### PR TITLE
Fix dialog opening mechanism #LMR-1859

### PR DESCRIPTION
**Pull request by livthomas**

### CODE REVIEW

File: src/app/dialog/dialog.service.ts [hunk 1]
```diff
@@ -40,34 +43,33 @@ import {Store} from '@ngrx/store';
...
 
-  public constructor(private router: Router, private store: Store<AppState>, private route: ActivatedRoute) {
+  private open$: Observable<boolean>;
+
+  public constructor(private router: Router, private store: Store<AppState>) {
     if (environment.videoKey) {
       this.store.dispatch(new VideosAction.LoadVideos({videos: getAllVideos(), apiKey: environment.videoKey}));
     }
+
+    this.open$ = this.bindOpen();
```
Comment (kubedo8) [inline src/app/dialog/dialog.service.ts]: Do we need this property still there?

**Comment by kubedo8** [comment]
Did you try to create some projects? If not, please try to create project at least 3 times to ensure everything works properly.

File: src/app/dialog/dialog.service.ts (hunk 1 above)
Comment (livthomas) [reply src/app/dialog/dialog.service.ts]: No, but we might forget about these problems in a few months. I added it here in order to allow us to access dialog state in the future without implementing nasty workarounds as we did in the past.

Comment (kubedo8) [reply src/app/dialog/dialog.service.ts]: Ok

**Comment by livthomas** [comment]
You are right. There are still some problems. Do not merge this yet!

**Pull request merged**